### ISSUE TITLE
Extra + symbol getting added in input box

### ISSUE THREAD

**Issue by jhachandan1994** [pmattes/x3270#90]
Some of the input boxes have a + symbol added by default and if the symbol is removed terminal throws error code as 0011. whereas if i use ibm i emulator there is no + symbol present on that input box. any idea what might be causing a + symbol to be present on a input box. 

<img width="154" alt="Screenshot 2024-09-18 at 8 37 25 PM" src="https://github.com/user-attachments/assets/88cf7e33-3294-4a76-be9e-ae233040f410">
<img width="62" alt="Screenshot 2024-09-18 at 8 38 03 PM" src="https://github.com/user-attachments/assets/c6f1406c-d81d-4be1-918e-fedfe2b8775d">

**Comment by swhobbit** [comment]
Curious, as  long term user of x3270 I've never seen this.  What's the environment?

I suspect Paul will need a trace.  (For ALL the interesting problems, he always need the trace. :-) )

**Comment by pmattes** [comment]
Yes, I need a trace. You don't need to trace the whole session. Just navigate to the place where the + symbols appear, then turn on tracing, then turn it back off. Then post the trace file here.
